CDCgov/prime-reportstream
# Add performing facility to Message ID Details

This PR adds the `testingLabName` field to the response for `/api/message/<report_index>`

Sample Response (test data in my local env)
```json
{
  "id" : 16,
  "messageId" : "797620",
  "sender" : "simple_report.default",
  "submittedDate" : "2022-08-31T13:57:49.609785",
  "reportId" : "342a21b3-fa85-4b88-9717-49e55ead1205",
  "fileName" : null,
  "fileUrl" : "http://localhost:10000/devstoreaccount1/reports/receive%2Fsimple_report.default%2Fpdi-covid-19-342a21b3-fa85-4b88-9717-49e55ead1205-20220831135748.csv",
  "testingLabName" : "Any lab USA", // New value!
  "warnings" : [ ],
  "errors" : [ ],
  "receiverData" : [ ] // Omitted for brevity
}
```

Test Steps:
>**NOTE:** Test data in your local environment, and a local database explorer like DBeaver is necessary for locating a valid message ID in the covid metadata table.
1. Use the `/api/message/<report_index>` endpoint with a valid message ID to ensure the property is returned

## Changes
- Adds `testingLabName` as a string in the JSON response for the message details endpoint

## Checklist

### Testing

### Process

## Linked Issues
- Fixes #7850 

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
